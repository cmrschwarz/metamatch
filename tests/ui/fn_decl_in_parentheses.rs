use metamatch::unquote;
fn main() {
    unquote! {
        (fn foo() {}) + 3
    }
}
