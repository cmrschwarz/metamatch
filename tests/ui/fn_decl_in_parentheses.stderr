error: `fn` cannot occur within expression
 --> tests/ui/fn_decl_in_parentheses.rs:4:10
  |
4 |         (fn foo() {}) + 3
  |          ^^
